src/App.js

```diff
@@ -1,19 +1,31 @@
 import Clothing from './components/Clothing'
 import styled from 'styled-components/macro'
-import data from './mockup-data'
+import mockupData from './mockup-data'
 
 function App() {
+  let data
+
+  if (localStorage.getItem('localClothing')) {
+    data = JSON.parse(localStorage.getItem('localClothing'))
+  } else {
+    localStorage.setItem('localClothing', JSON.stringify(mockupData))
+    data = JSON.parse(localStorage.getItem('localClothing'))
+  }
+
   return (
     <StyledMain>
-      {data.map((data) => (
+      {data.map((item) => (
         <Clothing
-          key={data.id}
-          title={data.title}
-          color={data.color}
-          pattern={data.pattern}
-          material={data.material}
-          fitting={data.fitting}
-          imageUrl={data.imageUrl}
+          data={data}
```

Comment:
You are giving down the complete `data`. Do you need that in your Clothing component?